packages/scandipwa/package.json

```diff
@@ -32,7 +32,8 @@
         "themeAlias": "Source",
         "type": "theme",
         "extensions": {
-            "@scandipwa/m2-theme": true
+            "@scandipwa/m2-theme": true,
+            "@scandipwa/webpack-i18n-runtime": true
```

Comment:
Where is the dependency itself? :)

Reply:
Is it possible to add the dependency before the package itself is published to the `npm`? 🤔

Reply:
Just do it!

Reply:
I am surprised not to see package-lock in the project :D will add the dep now.